configure.ac

```diff
@@ -353,7 +353,7 @@ if test "x$CXXFLAGS_overridden" = "xno"; then
   AX_CHECK_COMPILE_FLAG([-Wformat],[WARN_CXXFLAGS="$WARN_CXXFLAGS -Wformat"],,[[$CXXFLAG_WERROR]])
   AX_CHECK_COMPILE_FLAG([-Wvla],[WARN_CXXFLAGS="$WARN_CXXFLAGS -Wvla"],,[[$CXXFLAG_WERROR]])
   AX_CHECK_COMPILE_FLAG([-Wswitch],[WARN_CXXFLAGS="$WARN_CXXFLAGS -Wswitch"],,[[$CXXFLAG_WERROR]])
-  AX_CHECK_COMPILE_FLAG([-Wformat-security],[WARN_CXXFLAGS="$WARN_CXXFLAGS -Wformat-security"],,[[$CXXFLAG_WERROR]])
+  AX_CHECK_COMPILE_FLAG([-Wformat -Wformat-security],[WARN_CXXFLAGS="$WARN_CXXFLAGS -Wformat-security"],,[[$CXXFLAG_WERROR]])
```

Comment:
I think it is counter intuitive to test whether flag `X` is supported and if yes, then compile with flag `Y`. IMO it is better to always have `X` equal to `Y`. In this particular case the code would depend on `-Wformat` being already set from somewhere else and would break if that is removed.

Having that in mind, would it be better to test whether `-Wformat -Wformat-security` is supported and if yes, then enable `-Wformat -Wformat-security`? We would end up with `-Wformat` being mentioned twice in the flags, but there is no downside to that?